distributable/schema.graphql

```diff
@@ -243,6 +243,26 @@ type CumulativeVolumePerCurrencyPair @entity {
     collateralCurrency: Currency
 }
 
+type PooledAmount {
+    amount: BigInt!
+    amountHuman: BigDecimal
+    token: PooledToken!
+    # TODO: check if this should be Currency?
```

Comment:
@peterslany , I could use your input here. I believe using `PooledToken` instead of `Currency` makes sense here, but wanted to double check that with you.

For reference, `PooledToken` is defined as
`union PooledToken = NativeToken | ForeignAsset | StableLpToken`

Reply:
That is technically correct @peterslany, see the definition [here](https://github.com/interlay/interbtc/blob/18efc1e9194a8591713b9885799a6deb6c0392f2/primitives/src/lib.rs#L483-L487). Although I am not sure if this would be extended to support lend tokens for example.